client/my-sites/people/people-invites/index.jsx

```diff
@@ -25,39 +27,62 @@ class PeopleInvites extends React.PureComponent {
 		site: PropTypes.object.isRequired,
 	};
 
-	renderInvite( invite ) {
+	renderInvite = invite => {
 		// If an invite was sent to a WP.com user, the invite object will have
 		// either a display name (if set) or the WP.com username.  Invites can
 		// also be sent to any email address, in which case the other details
 		// will not be set (but the server will still set `avatar_URL` based on
 		// the email).
```

Comment:
This comment block should be moved to the appropriate place(s) in `PeopleListItem` (unless the code for each of these cases is obvious enough not to need comments).